packages/genesys-spark-components/src/components/stable/gux-popover/gux-popover.tsx

```diff
@@ -86,6 +86,13 @@ export class GuxPopover {
     }
   }
 
+  @Listen('focusout')
```

Comment:
The existing behavior is that if both `close-on-click-outside` and `display-dismiss-button`  are set to `true`, then clicking outside of the popover does cause the popover to close. With this change added to close on focusout, this behavior is no longer preserved (clicking outside is focusing out) I think this could be considered a breaking change?


![image](https://github.com/MyPureCloud/genesys-spark/assets/82100934/30c41e30-8140-494d-996e-1879f2e9a631)

Reply:
@katie-bobbe-genesys great catch. I pushed code to try and handle this scenario. The new code I wrote should handle these scenarios:

1. `close-on-click-outside` is `false` and `display-dismiss-button` is `true`
- User tabs out of popover -> popover closes
- User clicks out of popover -> popover does not close
2. `close-on-click-outside` is `true`
- User tabs out of popover -> popover closes
- User clicks out of popover -> popover closes
3. `close-on-click-outside` is `false` and `display-dismiss-button` is `false`
- User tabs out of popover -> popover closes
- User clicks out of popover -> popover closes

Reply:
```
close-on-click-outside is false and display-dismiss-button is true
User tabs out of popover -> popover closes
User clicks out of popover -> popover does not close
```
I may be mistaken, but I was thinking the behavior should be that if `close-on-click-outside` is false and `display-dismiss-button` is true:
User tabs out of popover -> popover does not close
User clicks out of popover -> popover does not close

If this is the case, I believe we just need to check `if !close-on-click-outside and display-dismiss-button` then do not close on focus out.  In all other cases, we want to close on focus out. I don't think we need to add the check for whether `tab` is pressed or not. Let me know if I'm missing something

Reply:
@katie-bobbe-genesys  Great catch! I pushed up a fix for this just now.

Reply:
> ```
> close-on-click-outside is false and display-dismiss-button is true
> User tabs out of popover -> popover closes
> User clicks out of popover -> popover does not close
> ```
> 
> I may be mistaken, but I was thinking the behavior should be that if `close-on-click-outside` is false and `display-dismiss-button` is true: User tabs out of popover -> popover does not close User clicks out of popover -> popover does not close
> 
> If this is the case, I believe we just need to check `if !close-on-click-outside and display-dismiss-button` then do not close on focus out. In all other cases, we want to close on focus out. I don't think we need to add the check for whether `tab` is pressed or not. Let me know if I'm missing something

@katie-bobbe-genesys This is now ready for review again. Thank you.